java/src/jmri/jmrit/ctc/NBHSignalMast.java

```diff
@@ -45,11 +45,15 @@ protected NBHSignalMast(String signal) {
             SignalMast signalMast = InstanceManager.getDefault(jmri.SignalMastManager.class).getSignalMast(signal);
             if (signalMast != null) {
                 _mNamedBeanHandleSignalMast = InstanceManager.getDefault(NamedBeanHandleManager.class).getNamedBeanHandle(signal, signalMast);
-                _mDangerAppearance = getAppearanceMap().getSpecificAppearance(SignalAppearanceMap.DANGER);
+//  Do this to get around a "bug" in the compiler.  When I used "_mDangerAppearance" instead of "temp",
```

Comment:
There is no "bug" in the compiler; when a field is marked __final__ it can only be written to _once_, so it did not "wrongly" complain. Do not include comments describing the compiler acting precisely as instructed to as descriptions of it working incorrectly.

Reply:
Removed.  I beg forgiveness in advance, I may have some of these in my code elsewhere, but I have no way of finding them.  Sometimes I just need to vent and it goes into the code, and I forget to remove them.